rb/lib/selenium/webdriver/remote/response.rb

```diff
@@ -28,20 +28,17 @@ class Response
         attr_reader :code, :payload
 
         def initialize(code, payload = nil)
-          @code    = code
+          @code = code
           @payload = payload || {}
 
           assert_ok
         end
 
         def error
-          error, message, backtrace = process_error
+          error, message = process_error
           klass = Error.for_error(error) || return
-
           ex = klass.new(message)
-          ex.set_backtrace(caller)
-          add_backtrace ex, backtrace
-
+          add_cause(ex, error)
```

Comment:
The cause needs to include a backtrace from the server.

Reply:
Now the cause includes backtrace